### DISCUSSION TITLE
sumatra pdf does not display characters from a local language(hindi) ebook file

### DISCUSSION THREAD

**Discussion by kinkblot** [sumatrapdfreader/sumatrapdf#3575] [Q&A]
the entire eBook is blank. the eBook is attached. i have tried other softwares like moon reader pro, and Read aloud. file works in both.

after some searching, it looks like a problem with fonts structuring. but I do not know how to do that. 
![Sumatra Render](https://github.com/sumatrapdfreader/sumatrapdf/assets/50849787/f3313003-fdac-48da-a96d-a78cd4db3d3d)
[Awara Bheed Ke Khatare (Hindi Edition).zip](https://github.com/sumatrapdfreader/sumatrapdf/files/12024109/Awara.Bheed.Ke.Khatare.Hindi.Edition.zip)

**Comment by GitHubRulesOK** [comment]
Thanks for including sample

It actually works well in MuPDF so its a SumatraPDF issue
 for tracking see https://github.com/sumatrapdfreader/sumatrapdf/issues/3576

**Comment by Hashemian01** [comment]
I have the exact same issue with Persian text. It used to be ok in 3.4.6
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/48093461/ea88e5f4-43aa-4e08-af01-c610421bf988)
